packages/api/drizzle/0042_migrate_auth_data.sql

```
@@ -0,0 +1,40 @@
+-- Migrate auth data and add temp UUID columns for foreign keys
+
+-- Migrate credential (email+password) accounts
+INSERT INTO "account" ("id", "account_id", "provider_id", "user_id", "password", "created_at", "updated_at")
+SELECT
+  gen_random_uuid()::text,
+  u."new_id",
+  'credential',
+  u."new_id",
```

SQLFluff (4.1.0)

[error] Reuse of column alias '"new_id"' from line 7.

(AL08)